app/views/equipment_models/show.html.erb

```diff
@@ -147,6 +148,33 @@
        <p><%= Requirement.list_requirement_admins(current_user, @equipment_model) %></p>
   </section>
 <% end %>
+<% if can? :manage, EquipmentModel %>
+  <section id="pending_reservations">
+      <h2>Pending Reservations</h2><hr />
+
+      <div class="row">
+        <div class="col-md-3">
+          <h4>Today</h4>
+        </div>
+        <div class="col-md-3">
+          <h4>Next 7 Day</h4>
+        </div>
+      </div>
+      <div class="row giant-numbers">
+        <div class="col-md-3">
+          <%= @pending.future.count %>
```

Comment:
Yeah my bad on that, I think I switched it when I changed the scopes for some reason.